### PR TITLE
[plugin.audio.itvrs_radio_ice] v1.0.3

### PR DESCRIPTION
**Pull request by remichie**
### Description
ITVRS Radio audio Add-on

v1.0.3 (20-8-2019)

dropped the usage of script.module.neverwise

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by remichie** [comment]
@Rechi, Any chance you can review this.

**Comment by Lunatixz** [comment]
LGTM

File: plugin.audio.itvrs_radio_ice/addon.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+along, and dance when nobody's watching!
+Our music is deep and broad,
+and the sound we create is unique.
+Just call it awesome.</description>
+  <language>en</language> 
+    <platform>all</platform>
+    <source>https://github.com/remichie/repo-plugins/tree/leia/plugin.audio.itvrs_radio_ice</source>
+    <license>GPL-2.0</license>
+  <news>
+v0.0.3  (20-8-2019)
```
Comment (enen92) [inline plugin.audio.itvrs_radio_ice/addon.xml]: Please keep news only for the last version in future updates

File: plugin.audio.itvrs_radio_ice/default.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import sys
+import xbmcgui
+import xbmcplugin
+
+addon_handle = int(sys.argv[1])
+
+xbmcplugin.setContent(addon_handle, 'audio')
+
+url = 'http://stream.zeno.fm/3611cwn8mbruv'
+li = xbmcgui.ListItem(label='ITVRS - Radio Ice', iconImage='DefaultAudio.png')
```
Comment (enen92) [inline plugin.audio.itvrs_radio_ice/default.py]: Please don't use `iconImage` it will break for Matrix. Use `listitem.setArt` only.

**Comment by Razzeee** [comment]
Solves https://github.com/xbmc/repo-plugins/pull/2515

**Comment by enen92** [comment]
ping @remichie

**Comment by basrieter** [comment]
Closing due to lack of response. If this is still valid PR, please reopen it.

**Pull request closed without merging**